examples/quasiparticle/cross_cloud/cross_cloud_storage.py

```diff
@@ -0,0 +1,57 @@
+from pcf.particle.aws.s3.s3_bucket import S3Bucket
+from pcf.core import State
+from pcf.quasiparticle.cross_cloud.cross_cloud_storage.cross_cloud_storage import CrossCloudStorage
+import os
+import sys
+
+# Edit example json to work in your account
+cross_cloud_storage_example = {
+    "pcf_name": "cross_cloud_storage", # Required
+    "flavor": "cross_cloud_storage",
+    "particles": [{
```

Comment:
you shouldnt have to define both particles. you should just specify the storage name and it will create the particles behind the scenes.

Reply:
should be all thats required i think. 

`cross_cloud_storage_example = {
    "pcf_name": "cross_cloud_storage", # Required
    "flavor": "cross_cloud_storage"
}`

Reply:
I had it separate because the users might want to specify things like tags or other custom_configs for different cloud platforms. Maybe its fine tho. What do you think? @anovis